motionplan/motionPlanner.go

```diff
@@ -569,9 +571,10 @@ func CheckPlan(
 
 			modifiedSegment := &ik.State{Frame: sf, Position: poseInPath}
```

Comment:
Was tripping over this, can you change this to be modifiedState instead?  Its not actually a segment and that confused me

Reply:
Done!